src/views/Home/Home.js

```diff
@@ -7,16 +7,16 @@ import './Home.css';
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
-  const [title, setTitle] = userState('testTitle');
-  const [subTitle, setSubTitle] = userState('testSubTitle');
-  const [font, setFont] = userState('testFont');
-  const [align, setAlign] = userState('testAlign');
-  const [text, setText] = userState('testText');
+  const [title, setTitle] = useState('');
+  const [subTitle, setSubTitle] = useState('');
+  const [font, setFont] = useState('');
+  const [align, setAlign] = useState('testAlign');
```

Comment:
Excellent job Sydnie, I hope this lab felt good! Just an FYI, here we can set our default alignment using useState('center'). It may also need a value={align} on the input element where we call setAlign in <Editor /> for the default to work appropriately.